tests/installation/welcome.pm

```diff
@@ -95,14 +95,14 @@ sub run {
         # In upgrade mode, there is no product list shown in welcome screen
         unless (check_var('ARCH', 's390x') || get_var('UPGRADE')) {
             assert_screen('select-product');
-            my %hotkey = (
-                sles     => 's',
-                sled     => 'u',
-                sles4sap => get_var('OFW') ? 'u' : 'i',
-                hpc      => check_var('ARCH', 'x86_64') ? 'x' : 'u'
-            );
             my $product = get_required_var('SLE_PRODUCT');
-            send_key 'alt-' . $hotkey{$product};
```

Comment:
Please recover this behavior.

We wanted to test **shorcuts** and **clicks**, so in GUI we test using clicks and in textmode we test the shorcuts. Use only **send_key_until_needlematch** when there isn't shorcut available for that action (eg. navigating a list menu)

If you remove this shorcut behavior, we lost coverage of that behavior on the yast installation ;)